shard.yml

```diff
@@ -54,6 +54,7 @@ dependencies:
     github: askn/spinner
     version: ~> 0.1.1
 
+development_dependencies:
```

Comment:
There is two branches of development dependencies I we can group all Dev deos into one